packages/node_modules/@cerebral/fluent/src/paths.ts

```diff
@@ -1,4 +1,4 @@
-const pathLambdaRegex = /function\s*\((\w)\)\s*{\s*return\s*\1[.]?([\w.[\]]+);\s*}/g
+const pathLambdaRegex = /function\s*\((\w)\)\s*{\s*return\s*\1[.]?([\w.[\]]+);*\s*}/g
```

Comment:
I think to make it optional a question mark ("?") instead an asterisk ("*") would be correct.

Reply:
That is a good point, asterisk would have allowed 0 or many which is not incorrect but it would be highly unlikely so I have updated it to a question mark.